README.md

```diff
@@ -37,6 +35,28 @@ Install all javascript components using bower
 $ bower update
 ``
 
+Create a remote to pull updates from upstream
+
+``
+$ git remote add upstream https://github.com/e-mission/e-mission-phone.git
+``
+
+``
+$ npm install
```

Comment:
nit: I don't think that this is technically required. You had the issues with the glob, but that was before you pulled from upstream. I have never had to do this to get it to work, and I've pulled a fresh repo from upstream recently to get it to work.